### PR TITLE
ORM service is optional

### PR DESCRIPTION
**Pull request by mustanggb**
When using pair history ORM is now optional not required.

Removing this should allow MongoDB users to use pair history without having the ORM package installed.

### CODE REVIEW

**Comment by johanwilfer** [comment]
Thanks for the patch @mustanggb !

Did you also test this using Doctrine ORM and that still works?

**Comment by mustanggb** [comment]
I'd hold off on this for now, I've not had a chance to look into the failures etc.

If anyone else wants to investigate please do.

**Comment by johanwilfer** [comment]
@mustanggb I see the tests about MongoDB also fails in master, as this contribution came from you could you have a look at this?

**Comment by mustanggb** [comment]
@johanwilfer 
Looks like it's a bug introduced by mongodb-odm 2.8.0 (https://github.com/doctrine/mongodb-odm/pull/2630), I believe it should be fixed in https://github.com/doctrine/mongodb-odm/pull/2671.

Do you want to wait for the fix, or we can temporarily work around by requiring `<2.8` (or similar).

**Comment by johanwilfer** [comment]
I think the temp workaround sounds good, that way we get tests in master to not fail again.

If you want/can do it in this branch it should stop to fail and then we can merge this if that sounds good?

**Comment by mustanggb** [comment]
Done, unless you want them squashing into one commit.

I know GitLab supports that on merge/pull, not sure if GitHub does.

Alternatively keeping them separate would also for easy revert when the workaround is no longer needed, I'll let it up to you.

**Review by johanwilfer — APPROVED** [review]

**Comment by johanwilfer** [comment]
Looks great!

And yeah, Github supports squashing of merged branches into one commit (this is how it is setup in the repro currently):
image

On a personal note I perfer individual commits in the feature branch to follow the process, and to squash on merge (to follow the PR).

**Pull request merged**
